src/pages/product-page/Product.jsx

```diff
@@ -1,189 +1,50 @@
-                      add to cart
-                    </button>
-                    <button className="card-wishlist btn btn-outline">
-                      wishlist
-                    </button>
-                  </div>
-                </div>
-              </div>
-            </li>
-
-            <li>
-              <div className="card card-vrt">
-                <div className="card-img-container ">
-                  <img className="card-img" src={tracksuit} alt="track-suit" />
-                </div>
-                <div className="card-desc">
-                  <h3>Track suit</h3>
-                  <p>by H&M</p>
-                  <span className="card-price">Rs 499</span>
-                  <del className="card-mrp">Rs 999</del>
-                  <div className="card-btns ">
-                    <button className="card-cart btn btn-primary ">
-                      add to cart
-                    </button>
-                    <button className="card-wishlist btn btn-outline">
-                      wishlist
-                    </button>
-                  </div>
-                </div>
-              </div>
-            </li>
-
-            <li>
-              <div className="card card-vrt">
-                <div className="card-img-container ">
-                  <img className="card-img" src={tracksuit} alt="track-suit" />
-                </div>
-                <div className="card-desc">
-                  <h3>Track suit</h3>
-                  <p>by H&M</p>
-                  <span className="card-price">Rs 499</span>
-                  <del className="card-mrp">Rs 999</del>
-                  <div className="card-btns ">
-                    <button className="card-cart btn btn-primary ">
-                      add to cart
-                    </button>
-                    <button className="card-wishlist btn btn-outline">
-                      wishlist
-                    </button>
-                  </div>
-                </div>
-              </div>
-            </li>
-
-            <li>
-              <div className="card card-vrt">
-                <div className="card-img-container ">
-                  <img className="card-img" src={tracksuit} alt="track-suit" />
-                </div>
-                <div className="card-desc">
-                  <h3>Track suit</h3>
-                  <p>by H&M</p>
-                  <span className="card-price">Rs 499</span>
-                  <del className="card-mrp">Rs 999</del>
-                  <div className="card-btns ">
-                    <button className="card-cart btn btn-primary ">
-                      add to cart
-                    </button>
-                    <button className="card-wishlist btn btn-outline">
-                      wishlist
-                    </button>
-                  </div>
-                </div>
-              </div>
-            </li>
+            {getSortedArr.map((item) => {
```

Comment:
Instead of writing `item.src` `item.title`. You can destructure `item` here.

Reply:
sure, will make this change